doc/dev/kernel.xml

```diff
@@ -863,9 +863,10 @@ $ gap -b
 GAP4, Version: 4.dev of today, ....
 gap> LoadDynamicModule("./foo.so");
 gap> Print(foo);
-function ( <<arg-1>> )
-    <<compiled code>>
-endgap> foo(3);
+function ( x )
```

Comment:
Sorry, I can't check whether gap actually outputs "function ( x )" or "function ( <<arg-1>> )"

Reply:
Actually, I did it by following the instructions right there in the file -- when you say you can't check it, do you mean those instructions didn't work for you, or that you simply did not run them (which is fine, of course! Just in case there were problems with the instructions, I'd like to address them, too)

Reply:
No the instruction where fine, but for some reasons that I didn't want to invastigate my gap-master-version gives an error when I try to `LoadDynamicModule` and my gap-10.0-version still returns `function ( <<arg-1>> )`.